### PR TITLE
RDoc-1965 -copied the files that should bubble up from 5.0V

### PR DESCRIPTION
**Pull request by Drorlev**
the "simple files"

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+{CODE:java loading_entities_5_0@ClientApi\Session\LoadingEntities.java /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **startsWith** | String | prefix for which documents should be streamed |
+| **matches** | String | pipe ('&#124;') separated values for which document IDs should be matched ('?' any single character, '*' any characters) |
+| **start** | int | number of documents that should be skipped  |
+| **pageSize** | int | maximum number of documents that will be retrieved |
+| **skipAfter** | String | skip document fetching until a given ID is found and returns documents after that ID (default: `null`) |
+| Reference streamQueryStats (out parameter) | Information about the streaming query (amount of results, which index was queried, etc.) |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown:116]: please check how we marked reference (out params) in other java docs. In general there isn't output parameters in java  - we fake them using Reference class.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+| Return Value | |
+| ------------- | ----- |
+| CloseableIterator<StreamResult<T>> | Iterator with entities. |
+| streamQueryStats (out parameter) | Information about the streaming query (amount of results, which index was queried, etc.) |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown:121]: same here

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+| **OperateOnTypes** | `DatabaseItemType` | Indicates what should be exported. Default: `Indexes`, `Documents`, `RevisionDocuments`, `Conflicts`, `DatabaseRecord`, `Identities`, `CompareExchange`, `Subscriptions` |
+| **OperateOnDatabaseRecordTypes** | `DatabaseRecordItemType` | Indicates what should be exported from database record. Default: `Client`, `ConflictSolverConfig`, `Expiration`, `ExternalReplications`, `PeriodicBackups`, `RavenConnectionStrings`, `RavenEtls`, `Revisions`, `SqlConnectionStrings`, `Sorters`, `SqlEtls`, `HubPullReplications`, `SinkPullReplications` |
+| **IncludeExpired** | `boolean` | Should expired documents be included in the export. Default: `true` |
+| **RemoveAnalyzers** | `boolean` | Should analyzers be removed from Indexes. Default: `false` |
+| **TransformScript** | `String` | JavaScript-based script applied to every exported document. Read more [here](../../client-api/smuggler/what-is-smuggler#transformscript). |
+| **MaxStepsForTransformScript** | `int` | Maximum number of steps that transform script can process before failing. Default: 10000 |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:41]: check if we usually lower case those property names in java docs

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+{PANEL:Import}
+
+### Syntax
+
+{CODE:java import_syntax@ClientApi\Smuggler\WhatIsSmuggler.java /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **options** | `DatabaseSmugglerImportOptions` | Options that will be used during the import. Read more [here](../../client-api/smuggler/what-is-smuggler#databasesmugglerimportoptions). |
+| **stream** | `Stream` | Stream with data to import |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:58]: Stream in java? Maybe InputStream?

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+| Return Value | | 
+| ------------- | ----- |
+| `Operation` | Instance of Operation-class which gives you an ability to wait for the operation to complete and subscribe to operation progress events |
+
+### DatabaseSmugglerImportOptions
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **Collections** | `List<String>` | List specific of collections to import. If empty then all collections will be imported. Default: `empty` |
+| **OperateOnTypes** | `DatabaseItemType` | Indicates what should be imported. Default: `Indexes`, `Documents`, `RevisionDocuments`, `Conflicts`, `DatabaseRecord`, `Identities`, `CompareExchange`, `Subscriptions` |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:70]: this is enum no use casing like: SQL_ETLS, etc.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+| **OperateOnDatabaseRecordTypes** | `DatabaseRecordItemType` | Indicates what should be exported from database record. Default: `Client`, `ConflictSolverConfig`, `Expiration`, `ExternalReplications`, `PeriodicBackups`, `RavenConnectionStrings`, `RavenEtls`, `Revisions`, `SqlConnectionStrings`, `Sorters`, `SqlEtls`, `HubPullReplications`, `SinkPullReplications` |
+| **IncludeExpired** | `boolean` | Should expired documents be imported. Default: `true` |
+| **RemoveAnalyzers** | `boolean` | Should analyzers be removed from Indexes. Default: `false` |
+| **TransformScript** | `String` | JavaScript-based script applied to every imported document. Read more [here](../../client-api/smuggler/what-is-smuggler#transformscript). |
+| **MaxStepsForTransformScript** | `int` | Maximum number of steps that transform script can process before failing. Default: 10000 |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:75]: lower case those props? (please check)

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/attachments/indexing.java.markdown [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Indexes: Indexing Attachments
+
+The `AttachmentsFor` method returns a list of 
+[attachments](../document-extensions/attachments/what-are-attachments) 
+in a given document as well as basic information like `Name` or `Size`
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/document-extensions/attachments/indexing.java.markdown:5]: name, size -> lower case

File: Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.java.markdown [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+{CODE-TAB:java:JavaScript indexing_related_documents_5@Indexes\JavaScript.java /}
+{CODE-TABS/}
+
+{CODE:java indexing_related_documents_8@Indexes\IndexingRelatedDocuments.java /}
+
+## Remarks
+
+{INFO Indexes are updated automatically when related documents change. /}
+
+{WARNING Using the `LoadDocument` adds a loaded document to the tracking list. This may cause very expensive calculations to occur, especially when multiple documents are tracking the same document. /}
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.java.markdown:41]: lower case?

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-analyzers.java.markdown [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+- your DLL needs to be placed next to RavenDB binaries (note it needs to be compatible with .NET Core 2.0 e.g. .NET Standard 2.0 assembly)
+- the fully qualified name needs to be specified for an indexing field that is going to be tokenized by the analyzer
+
+{CODE analyzers_6@Indexes\Analyzers.cs /}
+
+## Manipulating Field Indexing Behavior
+
+By default, each indexed field is analyzed using the `LowerCaseKeywordAnalyzer` which indexes a field as a single, lower cased term.
+
+This behavior can be changed by turning off the field analysis (setting the `FieldIndexing` option for this field to `Exact`). This causes all the properties to be treated as a single token and the matches must be exact (case sensitive), similarly to using the `KeywordAnalyzer` on this field.
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/using-analyzers.java.markdown:89]: EXACT

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+| **value** | `Object` | Value of the dynamic field |
+| **stored** | `boolean` | Sets [FieldStorage](../indexes/storing-data-in-index). By default value is set to `false` which equals to `FieldStorage.No`. |
+| **analyzed** | `boolean` | Sets [FieldIndexing](../indexes/using-analyzers).<br/><br/>Values:<br/>`null` - `FieldIndexing.Default` (set by overloads without this 'parameter')<br/>`false` - `FieldIndexing.Exact`<br/>`true` - `FieldIndexing.Search` |
+| **options** | `CreateFieldOptions` | Dynamic field options |
+
+### Options
+
+| CreateFieldOptions | | |
+| ------------- | ------------- | ----- |
+| **Storage** | `FieldStorage?` | More information about storing data in index can be found [here](../indexes/storing-data-in-index). |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:42]: FieldStorage?

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+| **Indexing** | `FieldIndexing?` | More information about analyzers in index can be found [here](../indexes/using-analyzers). |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:43]: FieldIndexing?

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+| **TermVector** | `FieldTermVector?` | More information about term vectors in index can be found [here](../indexes/using-term-vectors). |
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:44]: FieldTermVector?

there isn't concept of nullable enums in java

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/DataSubscriptions/DataSubscriptions.java [hunk 13]
```diff
@@ -0,0 +1,637 @@
...
+        <T> String create(Class<T> clazz, SubscriptionCreationOptions options);
+
+        <T> String create(Class<T> clazz, SubscriptionCreationOptions options, String database);
+
+        <T> String createForRevisions(Class<T> clazz);
+
+        <T> String createForRevisions(Class<T> clazz, SubscriptionCreationOptions options);
+
+        <T> String createForRevisions(Class<T> clazz, SubscriptionCreationOptions options, String database);
+        //endregion
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/DataSubscriptions/DataSubscriptions.java:41]: I assume you copied that directly from java class/inteface right?

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java [hunk 14]
```diff
@@ -0,0 +1,221 @@
...
+    }
+
+    public void examples() {
+        try (IDocumentStore store = new DocumentStore()) {
+            try (IDocumentSession session = store.openSession()) {
+                //region projections_1
+                // request name, city and country for all entities from 'Companies' collection
+                QueryData queryData = new QueryData(
+                    new String[] { "Name", "Address.city", "Address.country"},
+                    new String[] { "Name", "City", "Country"});
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:68]: I think it won't work as by default we use lower case letters in json mapping. Please run and test that, ok ?

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java [hunk 15]
```diff
@@ -0,0 +1,221 @@
...
+                    .query(Company.class)
+                    .selectFields(NameCityAndCountry.class, queryData)
+                    .toList();
+                //endregion
+            }
+
+            try (IDocumentSession session = store.openSession()) {
+                //region projections_2
+                QueryData queryData = new QueryData(new String[]{ "ShipTo", "Lines[].ProductName" },
+                    new String[]{"ShipTo", "Products"});
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:79]: same here

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java [hunk 16]
```diff
@@ -0,0 +1,221 @@
...
+                    .selectFields(ShipToAndProducts.class, queryData)
+                    .toList();
+                //endregion
+            }
+
+            try (IDocumentSession session = store.openSession()) {
+                //region projections_3
+                List<FullName> results = session.advanced().rawQuery(FullName.class, "from Employees as e " +
+                    "select {" +
+                    "    FullName : e.FirstName + \" \" + e.LastName " +
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:91]: and here

Comment (ml054) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:91]: and in general in this file... :)

Comment (ml054) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:91]: https://github.com/ravendb/docs/blob/master/Documentation/4.0/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java

I can see same thing in original file. Anyway it is worth to test that.

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+                //region import_syntax
+                public Operation importAsync(DatabaseSmugglerImportOptions options, String fromFile);
+                public Operation importAsync(DatabaseSmugglerImportOptions options, InputStream stream);
+                //endregion
+
+
+
+
+                //region import_example
+                Operation importOperation =  store.smuggler().importAsync(importOptions,"C:\\ravendb-exports\\Northwind.ravendbdump");
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java:66]: code formatting, remove double spaces, put comma after `,` etc.

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/DocumentExtensions/Attachments/IndexingAttachments.java [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+import net.ravendb.client.documents.operations.attachments.AttachmentName;
+import net.ravendb.client.documents.session.IDocumentSession;
+
+import java.util.List;
+
+public class IndexingAttachments {
+
+    private interface IFoo {
+        //region syntax
+        List<AttachmentName> AttachmentsFor(Object doc);
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/DocumentExtensions/Attachments/IndexingAttachments.java:16]: lower case ? In general please always copy method signatures from client's code

**Comment by ml054** [comment]
general note: looks like pom.xml is missing in 5.1 java (see 4.0 https://github.com/Drorlev/docs/blob/RDoc-1965_5.1Docs/Documentation/4.0/Samples/java/pom.xml) please configure this as maven project (you can reference 5.2 java client there)

File: Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown (hunk 1 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown:116]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown (hunk 2 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/session/loading-entities.java.markdown:121]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown (hunk 3 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:41]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown (hunk 4 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:58]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown (hunk 5 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:70]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown (hunk 6 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:75]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/attachments/indexing.java.markdown (hunk 7 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/document-extensions/attachments/indexing.java.markdown:5]: Done.

File: Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.java.markdown (hunk 8 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/indexing-related-documents.java.markdown:41]: I found it only like this `LoadDocument`.

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-analyzers.java.markdown (hunk 9 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/using-analyzers.java.markdown:89]: Done.

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/DataSubscriptions/DataSubscriptions.java (hunk 13 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/DataSubscriptions/DataSubscriptions.java:41]: Yes

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java (hunk 14 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:68]: It worked.

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java (hunk 15 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:79]: It worked.

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java (hunk 16 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToProjectQueryResults.java:91]: They are works

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown (hunk 10 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:42]: It is written like this also in C#
so maybe I didn't the right porting to java?

**Comment by Drorlev** [comment]
> general note: looks like pom.xml is missing in 5.1 java (see 4.0 https://github.com/Drorlev/docs/blob/RDoc-1965_5.1Docs/Documentation/4.0/Samples/java/pom.xml) please configure this as maven project (you can reference 5.2 java client there)

should I do only the dependency?

**Comment by ml054** [comment]
@Drorlev there isn't java 5.1, but I think you can copy pom to 5.1 and use 5.2 there. 

I know you have 5.1 mainly to make things easier with sync, but the question is do we want to display java 5.1 on website. Maybe we should move that directly to 5.2? It will be harder for you but easier for users?

**Comment by Drorlev** [comment]
> general note: looks like pom.xml is missing in 5.1 java (see 4.0 https://github.com/Drorlev/docs/blob/RDoc-1965_5.1Docs/Documentation/4.0/Samples/java/pom.xml) please configure this as maven project (you can reference 5.2 java client there)

Done.

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown (hunk 12 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:44]: how should I do the right porting to java?

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java (hunk 17 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java]: Done

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown [hunk 19]
```diff
@@ -0,0 +1,104 @@
+﻿# What is Smuggler
+
+Smuggler gives you the ability to export or import data from or to a database using JSON format. It is exposed via the `DocumentStore.Smuggler` property.
```
Comment (ml054) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:3]: DocumentStore.Smuggler` property. - there isn't properties in java. `smugger()` IIRC

File: Documentation/5.1/Samples/java/pom.xml [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+            <groupId>net.ravendb</groupId>
+            <artifactId>ravendb</artifactId>
+            <version>5.2.0</version>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <maven.compiler.source>16</maven.compiler.source>
+        <maven.compiler.target>16</maven.compiler.target>
+    </properties>
```
Comment (ml054) [inline Documentation/5.1/Samples/java/pom.xml:22]: could you copy pom from different version and change only version to 5.2 (there isn't 5.1 java release)
Texts like `javaTryPom` or `org.example` doesn't look good. :)

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java [hunk 21]
```diff
@@ -0,0 +1,63 @@
...
+                // each load will create a new Employee instance
+                Assert.assertNotSame(employee1, employee2);
+                 */
+                //region for_database
+                DatabaseSmuggler northwindSmuggler = store.smuggler().forDatabase("Northwind");
+                //endregion
+
+                DatabaseSmugglerExportOptions smugglerExportOptions =null;
+                exportOptions=null;
+                importOptions=null;
```
Comment (ml054) [inline Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java:35]: fix spacing around `=`

File: Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown (hunk 19 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:3]: What should be the equivalent method in java?

Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/smuggler/what-is-smuggler.java.markdown:3]: What are the methods that should be the equivalent in java ?

File: Documentation/5.1/Samples/java/pom.xml (hunk 20 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/pom.xml]: done

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown (hunk 11 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:43]: Done

File: Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown (hunk 12 above)
Comment (Drorlev) [reply Documentation/5.1/Raven.Documentation.Pages/indexes/using-dynamic-fields.java.markdown:44]: Done

File: Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java (hunk 21 above)
Comment (Drorlev) [reply Documentation/5.1/Samples/java/src/test/java/net/ravendb/ClientApi/Smuggler/WhatIsSmuggler.java:35]: Done

**Review by ml054 — APPROVED** [review]

**Pull request merged**